source/jormungandr/jormungandr/default_settings.py

```diff
@@ -24,6 +24,9 @@
 
 DISABLE_DATABASE = boolean(os.getenv('JORMUNGANDR_DISABLE_DATABASE', False))
 
+# Active the asynchronous ridesharing mode
+ASYNCHRONOUS_RIDESHARING = boolean(os.getenv('JORMUNGANDR_ASYNCHRONOUS_RIDESHARING', False))
```

Comment:
this default value is already here: https://github.com/CanalTP/navitia/pull/3280/files#diff-3eb516880819f9f6747981aefb7a07f6R175, you don't need this in the global setting...